### PR TITLE
Some tweaks for packageJsonLoader and testProjects

### PR DESCRIPTION
**Pull request by coreylight**

### CODE REVIEW

**Comment by coreylight** [comment]
@pselle tagging for review

File: package.json [hunk 1]
```diff
@@ -9,14 +9,18 @@
     "jest": "jest",
     "prepare": "npm run build",
     "prepublishOnly": "npm run build",
-    "test": "npm run eslint && npm run build && npm run jest",
+    "test": "npm run eslint && npm run build && npm run jest && npm run testIndividualProjects",
+    "testIndividualProjects": "npm run projectTestPackageJsonConfig",
```
Comment (pselle) [inline package.json:13]: It looks like you don't need this separate command anymore since it only calls another

File: src/config/util.js [hunk 2]
```diff
@@ -22,14 +20,22 @@ function getPackageConfig() {
  * Attempts a require and instantiation from a given string.
  */
 function requireFromString(src, args) {
+  /*eslint-disable camelcase, no-undef*/
+  /* webpack bug: https://github.com/webpack/webpack/issues/5939 */
```
Comment (pselle) [inline src/config/util.js:24]: This is a really good comment :+1:

File: testProjects/packageJsonConfig/handler.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import _ from 'lodash';
+
+const iopipe = require('./iopipe');
+
+describe('Using package.json iopipe configuration', () => {
+  beforeEach(() => {
+    delete process.env.IOPIPE_TOKEN;
```
Comment (pselle) [inline testProjects/packageJsonConfig/handler.test.js:7]: an enhancement to this (not necessarily this PR to the branch) would be to add a test covering the order preference (ex. when env var is set and config things are set and some code things are set etc.)

**Comment by kolanos** [comment]
@coreylight I wasn't assigned to review, but this LGTM.

File: package.json (hunk 1 above)
Comment (coreylight) [reply package.json:13]: I figure there will be more, as we support iopipe.rc down the line...

Comment (pselle) [reply package.json:13]: wfm

**Review by kolanos — APPROVED** [review]

**Comment by coreylight** [comment]
@pselle you down with this?

**Comment by pselle** [comment]
@coreylight I still hate the dependency addition but to make this possible ... so yes.

**Comment by pselle** [comment]
@coreylight @kolanos 👍  but I don't have write access to this branch to merge

**Comment by coreylight** [comment]
Yours to merge @kolanos

**Pull request merged**